modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionContext.java

```diff
@@ -156,6 +156,7 @@ public ColocationGroup target() {
      * @param exchangeId ExchangeId to find remote nodes for.
      * @return Remote nodes for given exchangeId.
      */
+    @Nullable
```

Comment:
if we annotated as Nullable let's mention in javadoc when null could be returned

Reply:
Added mention in javadoc, thnx.